### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by bnhowell** [projecthydra/sufia#3093]
Related to Sufia 7.2

1. Navigation: Home, About, Help and Contact navigation menu items have insufficient contrast for non-selected state: 4.22 (foreground color: #777777, background color: #f8f8f8). Should be 4.5:1 at a minimum.

All pages: Don't disable zooming

Home, About, Help pages: Skip to content link isn’t visible on focus. Skip link not apparent with VoiceOver or JAWS.

Contact Page: Skip to Content is announced with NVDA but does not work 

Create New Work page (Add files): The “+ Add files ...” control is not in the focus order. When arrowing to this control with NVDA, it is not announced as “clickable”. Instead, an off-page option “Browse” is exposed to screen readers, which is not as descriptive a label for the control, in my opinion. Perhaps “Upload file”? 

Create New Work page (Add files): "Uploading a file was problematic with JAWS. The option to look for a file is present, but once you choose a file, you have no confirmation that the file has been uploaded. The option to Save is not available, nor is the Cancel option.”

Create New Work page (Save Work Panel): The expanding radio button text + controls are problematic. The screen reader user is not alerted to the fact that there are controls in the radio button text that are actionable for some of the options. The radio buttons are also not part of a group. 

Create New Work page (Relationships tab) Relationships: “Collection ids” announced as “clickable”, but does not appear to be a control. List announced with screen reader, but it is not obvious how to select list item, other than tabbing out of list. Suggest: “Collection IDs”. May get better pronunciation with screen readers.

Create New Work page (Save Work Panel): Difficult with NVDA to discover the Requirements message because you must use Tab and Shift+Tab to move away from the “Collection ids” list and the “Visibility” radio buttons. So it jumps over intervening text.

**Comment by jcoyne** [comment]
Would you mind filing each issue as a separate ticket? I think several of these need discussion and it will be confusing talking about all of them together on one ticket.

**Comment by bnhowell** [comment]
@jcoyne I'll break these apart into individual issues.

**Comment by jcoyne** [comment]
These are now separate tickets.
